wiki/People/Beatmap_Nominators/en.md

```diff
@@ -34,11 +34,11 @@ Modders that aim to apply to the Beatmap Nominators must fulfil the following cr
 ### How the Required Minimum Score is Calculated
 + This depends on the total `Mod Score` an applicant achieved over the last months. This value is determined according to the following formula:
 
-+ `Kudosu Score` is determined according to the following formula:
+![Mod Score Formula](/wiki/shared/Modscore_new.png "Kudosu Score Formula")
```

Comment:
the image should use white text instead of black

Reply:
idk how to do this aaa